### PR TITLE
Add Portuguese geoid model GeodPT08

### PR DESCRIPTION
**Pull request by jjimenezshaw**
Geoid model GeodPT08 from https://www.dgterritorio.gov.pt/geodesia/modelo-geoide

Related PR in PROJ: https://github.com/OSGeo/PROJ/pull/4063

### CODE REVIEW

File: pt_dgt/pt_dgt_README.txt [hunk 1]
```diff
@@ -9,12 +9,12 @@ from [DG Territorio](http://www.dgterritorio.pt/)
 
 Grid transformation from Portuguese Lisbon Datum to ETRS89 in Portugal.
 
-*Source*: [DGT](http://www.dgterritorio.pt/cartografia_e_geodesia/geodesia/transformacao_de_coordenadas/grelhas_em_ntv2/)  
-*Format*: GeoTIFF converted from NTv2  
-*License*: [Creative Commons Attribution 4.0](http://creativecommons.org/licenses/by/4.0/)  
+*Source*: [DGT](http://www.dgterritorio.pt/cartografia_e_geodesia/geodesia/transformacao_de_coordenadas/grelhas_em_ntv2/)
```
Comment (rouault) [inline pt_dgt/pt_dgt_README.txt:12]: the 2 trailing spaces are intended for correct formatting in github

File: pt_dgt/build_GeodPT08.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            sh -c " \
+            # Call vertoffset_grid_to_gtiff-script
+            python3 ${PROJ_DATA_DIR}/grid_tools/vertoffset_grid_to_gtiff.py \
+            --type GEOGRAPHIC_TO_VERTICAL \
+            --parameter-name geoid_undulation \
+            --source-crs \"EPSG:4937\" \
+            --target-crs \"EPSG:5780\" \
+            --description \"ETRS89 (EPSG:4937) to Cascais height (EPSG:5780). Converted from $orig\" \
+            --area-of-use \"Portugal - mainland - onshore.\" \
+            --copyright \"Derived from work by Direção-Geral do Território (DGT+FCUL). CC-BY-4.0 https://creativecommons.org/licenses/by/4.0/\" \
```
Comment (rouault) [inline pt_dgt/build_GeodPT08.sh:34]: I'm not sure to remember the encodings of the Copyright TIFF tag. But let's keep it to ASCII for maximal compatiblity (pt_dgt_D73_ETRS89_geo.tif for example has TIFFTAG_COPYRIGHT=Derived from work by DGT. Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/)
```suggestion
            --copyright \"Derived from work by DGT+FCUL. CC-BY-4.0 https://creativecommons.org/licenses/by/4.0/\" \
```

**Review by rouault — COMMENTED** [review]
-

File: pt_dgt/pt_dgt_README.txt (hunk 1 above)
Comment (jjimenezshaw) [reply pt_dgt/pt_dgt_README.txt]: restored

File: pt_dgt/build_GeodPT08.sh (hunk 2 above)
Comment (jjimenezshaw) [reply pt_dgt/build_GeodPT08.sh]: done

**Pull request merged**